web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
+- Navigating to the metrics page presents users with an empty query field and a prompt to enter a query in the box
+
+![Metrics page with too many results to graph](img/metrics-full.png)
+- If a query cannot be graphed, the results should still be shown in the table below, but the graph field will be populated by an error message until the issue is resolved or the offending query is switched off.
```

Comment:
Should this have an inline notification style? Or maybe it's worth checking in on the latest PatternFly Dashboard "no data available" style guideline?

Reply:
do you know where those style guidelines might be?

Reply:
A related nit: this error message also refers to the visualization as a "graph", but the top-right button refers to it as a "chart". Also needs a period at the end of the second sentence (image below this one too).

And should the Namespace cells here include the NS resource badge?

Reply:
> do you know where those style guidelines might be?

@cshinn I checked in with @mceledonia and he pointed me to the PF4 guidelines for Empty State which should be applicable to any graph area as well. There is an example of a smaller card here: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/empty-state

Reply:
I was previously using the empty state without a title or icon, but I've added those back in to provide a bit more visual interest

Reply:
The empty state style introduced by https://github.com/openshift/console/pull/1653 has a gray background, so I guess we should use that here too?